src/main/webapp/scripts/searchmanagement/views/search.controller.js

```diff
@@ -358,12 +362,25 @@ angular.module('metadatamanagementApp').controller('SearchController',
       MessageBus.set('onDataPackageFilterChange', dataPackageFilter);
     }
 
+    function createPublicationsFilterObject(data) {
+      if (Principal.isAuthenticated()) { return null; }
+      var dataPackageFilter = {
+        'year': data.year.buckets,
+        'language': data.language.buckets
+      };
+      MessageBus.set('onDataPackageFilterChange', dataPackageFilter);
+    }
+
     function createTotalHitsObject(totalHitsInCurrentIndex,
       totalHitsInAdditionalIndex) {
       var totalHits = {};
       totalHits[$scope.searchParams.type] = totalHitsInCurrentIndex;
-      totalHits[$scope.searchParams.additionalSearchIndex] =
-        totalHitsInAdditionalIndex;
+      for (var index in $scope.searchParams.additionalSearchIndex) {
```

Comment:
Hier wäre ebenfalls eine reguläre For-Loop besser, wenn es um den reinen Index in dem Array geht.